src/features/add-ci-link.js

```diff
@@ -0,0 +1,44 @@
+import select from 'select-dom';
+import domify from '../libs/domify';
+import {getRepoURL} from '../libs/page-detect';
+
+// This var will be:
+// - undefined on first load
+// - a Promised dom element after a successful fetch
+// - false after a failed fetch
+let request;
+
+async function fetchStatus() {
+	const url = `${location.origin}/${getRepoURL()}/commits/`;
+	const dom = await fetch(url).then(r => r.text()).then(domify);
```

Comment:
@bfred-it This could be:

```js
const response = await fetch(url);
const dom = domify(await response.text());
```

And here too:

```
src/content.js
88:	domLoaded.then(onDomReady);

src/libs/synchronous-storage.js
30:		return get().then(value => {

src/libs/utils.js
36:	domLoaded.then(() => requestAnimationFrame(() => waiting.cancel()));

src/libs/api.js
3:	return fetch(api + endpoint).then(res => res.json());

src/features/linkify-branch-refs.js
33:	safeElementReady('.branch-name').then(el => {

src/features/show-names.js
16:		.then(res => res.text());

src/features/show-recently-pushed-branches.js
18:	}).then(res => res.text());

src/features/mark-unread.js
335:			}).then(storage => storage.unreadNotifications);

src/features/move-account-switcher-to-sidebar.js
5:	safeElementReady('.dashboard-sidebar').then(sidebar => {
```

Reply:
Done in #814